assets/js/phoenix_live_view.js

```diff
@@ -751,6 +753,17 @@ let DOM = {
         }
         DOM.discardError(toEl)
 
+        // equality check with input value
+        if (fromEl.nodeName == "INPUT") {
+          const fromCopy = fromEl.cloneNode(true)
+          // input value comes from property
+          fromCopy.setAttribute("value", fromCopy.value)
+          if (fromCopy.isEqualNode(toEl)) {
+            // avoid extra attribute change, fixes Chrome decimal erase bug
+            return false
```

Comment:
Would this `return false` for pretty much every input change?  Would it then be better to just let morphdom do its thing on `blur`?